bears/documentation/DocumentationStyleBear.py

```diff
@@ -0,0 +1,87 @@
+from coalib.bearlib.languages.documentation.DocumentationComment import (
+    DocumentationComment)
+from coalib.bearlib.languages.documentation.DocstyleDefinition import (
+    DocstyleDefinition)
+from coalib.bearlib.languages.documentation.DocumentationExtraction import (
```

Comment:
oh indeed it does not work :O
in fact you need to assign the imports yourself:
```python
import coalib.bearlib.languages.documentation as coaliub_documentation
DocumentationComment = coalib_documentation.DocumentationComment
# ...
```
But I'm not sure if it's that nicely, I would prefer normal imports^^